apps/dev/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Avoid using duplicate eslint configs, just update the root config if this is needed. We have a separate section for storefront/next stuff: https://github.com/digdir/designsystem/blob/a8cc97ebe61b9d82257276186b0a171705387a32/.eslintrc.js#L39